content/en/user-guide/aws/dms/index.md

```diff
@@ -0,0 +1,167 @@
+
+```sh
+************
+STARTING FULL LOAD FLOW
+************
+db endpoint: localhost:3306
+
+	Cleaning tables
+	Creating tables
+	Inserting data
+
+	Added the following authors
+[{'first_name': 'John', 'last_name': 'Doe'}]
+
+	Added the following accounts
+[{'account_balance': Decimal('1500.00'), 'name': 'Alice'}]
+
+	Added the following novels
+[{'author_id': 1, 'title': 'The Great Adventure'},
+ {'author_id': 1, 'title': 'Journey to the Stars'}]
+
+****Full Task 1****
+
+
+	Starting Full load task 1 a%
+Replication Task arn:aws:dms:us-east-1:000000000000:task:FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7 status: starting
+Waiting for task status stopped
+task='arn:aws:dms:us-east-1:000000000000:task:FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7' status='starting'
+task='arn:aws:dms:us-east-1:000000000000:task:FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7' status='stopped'
+
+	Kinesis events
+
+fetching Kinesis event
+Received: 6 events
+[{'control': {},
+  'metadata': {'operation': 'drop-table',
+               'partition-key-type': 'task-id',
+               'partition-key-value': 'FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7',
+               'record-type': 'control',
+               'schema-name': 'dms_sample',
+               'table-name': 'accounts',
+               'timestamp': '2024-05-23T19:17:33.126Z'},
+  'partition_key': 'FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7.dms_sample.accounts'},
+ {'control': {},
+  'metadata': {'operation': 'drop-table',
+               'partition-key-type': 'task-id',
+               'partition-key-value': 'FQWFF7YIZ4VGQHBIXCLI9FJTUUS17NSECIM0UR7',
+               'record-type': 'control',
+               'schema-name': 'dms_sample',
+               'table-name': 'authors',
+               'timestamp': '2024-05-23T19:17:33.128Z'},
+...
+...
+...
+```
+
+
+## Limitations
+
+For RDS MariaDB and RDS MySQL it is not yet possible to set custom db-parameters. 
+In order to make those databases work with `cdc` migration for DMS, some default db-parameters are changed upon start if the `ENABLE_DMS=1` flag is set:
+
+```sh
+binlog_checksum=NONE
+binlog_row_image=FULL
+binlog_format=ROW
+server_id=1
+log_bin=mysqld-bin
+```
+
+
+### Enum Values for CDC data events
+
+To support Enum values for CDC data events, you need to enable the database setting `BINLOG_ROW_METADATA=FULL`
+
+### Migration Type
+
+A replication task on LocalStack does currently only support `full-load` (migrate existing data) or `cdc` (replicate data changes only). 
+On AWS there is also a combination for those, which is not yet implemented on LocalStack.
+
+### ReplicationTaskSettings
+
+The `ReplicationTaskSettings` for a [replication task](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Tasks.CustomizingTasks.TaskSettings.html) only considers `BeforeImageSettings`, `FullLoadSettings.CommitRate` and `FullLoadSettings.TargetTablePrepMode`
+
+### Persistence
+
+Persistence on LocalStack for DMS is not supported.
```

Comment:
We have a new tag on our docs that showcases this information. We can remove this detail here.